pkg/rsyslog/Dockerfile.in

```diff
@@ -0,0 +1,20 @@
+# hadolint ignore=DL3006
+FROM GPTTOOLS_TAG as gpttools
+
+FROM alpine:3.8
+WORKDIR /
+COPY rsyslog@lists.adiscon.com-5a55e598.rsa.pub /etc/apk/keys/rsyslog@lists.adiscon.com-5a55e598.rsa.pub
+# hadolint ignore=DL3018
+RUN echo "http://alpine.rsyslog.com/3.7/stable" >> /etc/apk/repositories \
```

Comment:
I suspect you're doing this to be compatible with the version of alpine in gpttools -- so don't. Lets get rid of this dependency and then this line won't be needed.

Reply:
Actually rsyslog only has compiled binaries for alpine 3.7 version.
It looks like the binaries do not exist for other versions. I removed this repository addition and the build failed.
I also tried pointing to alpine 3.8 rsyslog repository and the build failed in this case also. Looks like this line will be required.